tests/queries/test_bulk_update.py

```diff
@@ -125,15 +125,17 @@ def test_update_custom_primary_key(self):
 
     def test_empty_objects(self):
         with self.assertNumQueries(0):
-            Note.objects.bulk_update([], ['note'])
+            count = Note.objects.bulk_update([], ['note'])
+            self.assertEqual(count, 0)
 
     def test_large_batch(self):
         Note.objects.bulk_create([
             Note(note=str(i), misc=str(i))
             for i in range(0, 2000)
         ])
         notes = list(Note.objects.all())
-        Note.objects.bulk_update(notes, ['note'])
+        count = Note.objects.bulk_update(notes, ['note'])
+        self.assertEqual(count, 2000)
```

Comment:
We should add an extra tests for duplicates, e.g. `bulk_update()` with the same object twice.